src/main/resources/io/jenkins/plugins/designlibrary/Layouts/index.jelly

```diff
@@ -0,0 +1,144 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:s="/lib/samples" xmlns:st="jelly:stapler">
+  <s:sample>
+    <div class="jdl-component-sample">
+      <div id="layout-preview" class="jdl-layout-preview jdl-layout-preview--two-column">
+        <div class="jdl-layout-preview__navigation" />
+        <div class="jdl-layout-preview__sidebar">
+          <div class="jdl-layout-preview__header" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+        </div>
+        <div class="jdl-layout-preview__content">
+          <div class="jdl-layout-preview__header" />
+          <div class="jdl-layout-preview__button" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+        </div>
+        <div class="jdl-layout-preview__cards">
+          <div class="jdl-layout-preview__header" />
+          <div class="jdl-layout-preview__button" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+        </div>
+      </div>
+    </div>
+
+    <div class="jenkins-tab-pane">
+      <h2 class="jenkins-tab-pane__title">Two column</h2>
+
+      <p class="jdl-leading-paragraph">
+        Two column layouts are for 'hub' style pages, pages that have child pages
+        for the user to navigate to.
+      </p>
+
+      <p class="jdl-paragraph">Examples of such pages include 'Dashboard', 'Project' and 'Plugin Manager'.</p>
```

Comment:
Isn't this view I'm reviewing (src/main/resources/io/jenkins/plugins/designlibrary/Layouts/index.jelly) another example? Or is this concept not yet usable in the design-library? Or when should we use a tab navigation like you did for {One column, Two Column, Fullscreen} and when the app-bar? This is not clear to me yet.

Reply:
> Isn't this view I'm reviewing (src/main/resources/io/jenkins/plugins/designlibrary/Layouts/index.jelly) another example?

Yep, I've replaced the 'Project' example with 'Design Library' 👍 

> Or when should we use a tab navigation like you did for {One column, Two Column, Fullscreen} and when the app-bar? This is not clear to me yet.

Headings/App bars should be on every page (eventually).

Tabs should be used if you have lots of data you need to display in a concise manner, however the data doesn't need to appear all at once. I'll add a 'Tabs' page on Design Library eventually with more guidance, there's still more thinking to do in that area though as the current tabs Jenkins uses isn't perfect atm.

Reply:
What I mean: the tabs and the app-bar in the side panel try to aim for the same goal: if you have too much data you can use either the links in the side panel or some tabs to show them. But when should I use tabs, and when the links in the side panel?

Reply:
Ah, nothing written up on that yet. I'd suggest using 'items in the sidebar' for when you want a full page replacement, e.g. going from Available plugins to Installed plugins. 

I feel that the tabs are more lightweight and should be used for smaller content changes, such as changing a section on a page (such as your code coverage page https://ci.jenkins.io/job/Plugins/job/analysis-model/job/master/1439/coverage/#overview)